packages/cli/src/workflows/create/Create.common.handler.ts

```diff
@@ -75,18 +76,19 @@ export default class CreateCommonHandler extends ZosmfBaseHandler {
         switch (sourceType) {
             case "dataset":
                 try {
-                    resp = await CreateWorkflow.createWorkflow(
-                        this.mSession,
-                        this.arguments.workflowName,
-                        this.arguments.dataSet,
-                        this.arguments.systemName,
-                        this.arguments.owner,
-                        this.arguments.variablesInputFile,
-                        this.arguments.variables,
-                        this.arguments.assignToOwner,
-                        this.arguments.accessType,
-                        this.arguments.deleteCompleted
-                    );
+                    resp = await CreateWorkflow.createWorkflow2({
```

Comment:
Even though the common handler is calling a different function (which usually translates into different/breaking behavior), I believe this is an exception as no existing end-user automation should be negatively impacted.

Reply:
@zFernand0 - perhaps we could have a guideline or contribution recommendation in the future on `v2` API naming standards.  In other cases, it seems like we introduced new method names instead of a incremental suffix.  No action needed, just observations.